test/integration/open-with/OpenWith.int.js

```diff
@@ -0,0 +1,22 @@
+// <reference types="Cypress" />
+
+describe('OpenWith', () => {
+    beforeEach(() => {
+        cy.server();
+    });
+
+    it('integrations will render when the dropdown is clicked', () => {
+        cy.route(
+            'GET',
+            '**/files/*/open_with_integrations',
+            'fixture:open_with_integrations/multipleIntegrations.json',
+        );
+
+        cy.visit('/ContentOpenWith');
```

Comment:
Consider moving this to `beforeEach` as it will reload the page for each test, giving you a fresh state.

Reply:
If you `visit` before you `route` the request will actually make it to the server. Since we're dynamically routing per test case I think we have to visit in each case.